ex02.exs

```diff
@@ -81,23 +74,24 @@ end
 defmodule Ex02 do
 
   def new_counter(initial_value \\ 0) do
-    # « your code goes here»
+	{:ok, counter} = Agent.start_link(fn->initial_value end)
+	counter
   end
 
   def next_value(agent) do
-    # « your code goes here»
+	value = Agent.get_and_update(agent, fn val -> {val, val +1} end}
```

Comment:
This line fails to compile for two different reasons:

~~~
** (SyntaxError) 2.exs:82: unexpected token: }. The "(" at line 82 is missing terminator ")"
~~~

* the trailing } should be )

I fixed that and I get

~~~
** (CompileError) 2.exs:82: "val +1" looks like a function call but there is a variable named "val", please use explicit parentheses or even spaces
~~~

If I fix both of these, then I get warnings about unused variables.  

# 0